docs/admin/index.rst

```diff
@@ -94,105 +141,75 @@ Once all the pieces are in place, you can deploy Renku with the following comman
 
-3. Post deployment configuration
+3. Verifying RenkuLab deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After Renku has been deployed you can make some post deployment configurations.
-For instance, make a user admin on GitLab.
+Check list:
 
-1. turn off automatic redirect to GitLab by setting redeploying with the value ``gitlab.oauth.autoSignIn: false``
-2. log in as the root user using the password from ``gitlab.password``
-3. modify any users you want to modify (e.g. to make them admin)
-4. turn the automatic redirect back on
+  * Helm command completed successfully.
+  * RenkuLab pods should be all running and jobs completed.
+  * RenkuLab pods should not show errors.
+  * Perform a quick check:
+    #. go to your RenkuLab instance domain
+    #. login with a valid user, you can register a new one (this will verify various authentication steps)
+    #. create a project (this will verify further authentication, templates and GitLab runner)
+    #. add a dataset (this will verify the core backend, LFS objects, and Knowledge Graph components work)
+    #. launch an environment (this will verify the notebooks component and GitLab image registry)
+  * You should be now able to follow "First steps"
 
-4. (Optional) Notebooks for anonymous users
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Logged-out users can be given the permission to launch interactive sessions from public notebooks. This feature is
-turned off by default. For details on how to enable this feature, see the dedicated section.
+If some Renku pods are not starting or present some errors please check the troubleshooting page.
 
 .. toctree::
    :maxdepth: 1
 
-   Enabling notebooks for anonymous users <anonymous-sessions>
-
-Verifying Renku
-------------------
-
-Check list:
-
-  - After a while (around 5 minutes) Renku pods should be all running.
-  - Verify logs of pods and check there are no errors.
-  - Perform a quick check:
-    1. go to your Renku instance domain
-    2. login with a valid user
-    3. create a project
-    4. launch a notebook
-  - You should be now able to follow "First steps"
-
-Additional configurations
--------------------------
-
-Privacy page
-~~~~~~~~~~~~
-
-The UI has a privacy page with a completely configurable content, suited for showing
-any policy/terms related information, like the `Privacy Policy Statement` or the
-`Terms of Use`.
-
-The content is read from a ``ConfigMap``. You need to configure the values in
-``ui.privacy.page`` to enable the feature and set the reference ConfigMap name and key.
-Both ``ui.privacy.enabled`` and ``ui.privacy.page.enabled`` need to be ``true`` for
-enabling the privacy page.
+   Troublesooting a Renku deployment <troubleshooting>
 
-.. note::
+**Note**. For deployments with existing GitLab and/or Keycloak, the post-install jobs need to be deleted manually in order for the Helm deployment to succeed.
```

Comment:
Are you sure? How does this work for our CI deployments?

Reply:
> Are you sure?

yes, we've been doing this for a couple of deployments with external GitLab and Keycloak in a regular basis. I _think_ the problem is for the keycloak deployment, which the CIs and personal deployments do have it enabled. Another issue is having the postgresql database externally, but this is a corner case and not covered in the minimal values script and template.
In any case, it seems clear that I should take out the GitLab part here.

Reply:
> I should take out the GitLab part here

Done

Reply:
so the post-install jobs run but never finish? If that's the case it sounds like we need to fix those.

Reply:
> so the post-install jobs run but never finish? If that's the case it sounds like we need to fix those.

They fail (iirc post-install keycloak job would never be able to connect to /auth since it has been deployed elsewhere) and as a consequence Helm thinks the deployment failed
```
Release "renku" failed: failed post-install: timed out waiting for the condition
```

So indeed we need to fix this. I created a PR that should fix this: https://github.com/SwissDataScienceCenter/renku/pull/1923 (indeed GitLab post-install job did not need this fix).

Reply:
Right, I was going to suggest a fix like that - but in that case I would remove this word of warning here since once the other PR is merged this will no longer happen. 👍

Reply:
> Right, I was going to suggest a fix like that - but in that case I would remove this word of warning here since once the other PR is merged this will no longer happen. +1

yes, this was the intention. I'm going to delete this warning. :+1: